rollout_service.rb

```diff
@@ -11,13 +11,18 @@ def authenticate!
         response = HTTParty.post('https://www.googleapis.com/oauth2/v3/tokeninfo',
                                  body: {id_token: params[:id_token]})
 
-        raise 'Bad response from server' if response.code != 200
+        raise "Bad response from server: #{response.inspect}" if response.code != 200
         response_body = JSON.parse(response.body)
 
         if $google_oauth_allowed_domain.present? && response_body['hd'] != $google_oauth_allowed_domain
           raise 'Unauthorized user, this domain is not allowed'
         end
 
+        allowed = (ENV['ALLOWED_EMAILS'] || '').split(',')
```

Comment:
Instead of accessing the ENV directly, set this list as a global.
See `$google_oauth_allowed_domain ` as a reference